mrchecker/Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+@Library('pdehnel-sharedlib@feature/devon4j') _
```

Comment:
please change to official name. can be found in sharedlib docu "ProductionLineTemplateLib"

Reply:
Ok